izanami-frontend/src/components/SearchComponant/SearchAction.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import Select, { components } from "react-select";
+import { customStyles } from "../../styles/reactSelect";
+import { SearchModalStatus, Option } from "../../utils/searchUtils";
+interface SearchActionProps {
+  tenant?: string;
+  allTenants?: string[];
+  modalStatus: { all: boolean; tenant?: string };
+  setModalStatus: (status: SearchModalStatus) => void;
+  filterOptions: { value: string; label: string }[];
+  setFilters: (filters: { value: string; label: string }[]) => void;
+}
+
+const CustomOption = (props: any) => {
+  return (
+    <components.Option {...props}>
+      <span style={{ marginRight: 1 }}>{props.data.icon}</span>
+      {props.data.label}
+    </components.Option>
+  );
+};
+
+export function SearchAction({
+  tenant,
+  allTenants,
+  modalStatus,
+  setModalStatus,
+  filterOptions,
+  setFilters,
+}: SearchActionProps) {
+  return (
+    <div
+      className="d-flex flex-row align-items-start my-1 gap-1"
+      role="group"
+      aria-label="Tenant selection"
+    >
+      <div className="d-flex flex-row gap-1 mb-2 mb-md-0">
+        <button
+          onClick={() => setModalStatus({ all: true })}
+          className={`btn btn-secondary${
+            modalStatus.all ? " search-form-button-selected" : ""
+          }`}
+          style={{ backgroundColor: "var(--color-blue) !important" }}
+          value="all"
+          aria-pressed={modalStatus.all}
+        >
+          All tenants
+        </button>
+        {tenant && tenant !== "all" && (
```

Comment:
This kind of magic string introduces potential bugs, for instance if you open search in a tenant named "all", tenant selection is bugged.